### PR TITLE
fix(gatsby-theme-bodiless): Properly prefix paths for bodiless gatsby…

### LINKED ISSUE #1501: Images are not correctly prefixed
## STR
1. Add a page with a gatsby image (eg any page on __cxstarter__ (logo is  a gatsby image)
2. FollowGatsby docs for path prefixing [here](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/path-prefix/)
3. Visit page in browser

## ER
Image is properly prefixed and renders correctly

## AR
Image is served from /static and does not render full resolution version.

### PR DESCRIPTION
**Pull request by wodenx**
## Changes
Applies path prefix (if active) to all gatsby image urls

## Related Issues
Fixes #1501

### CODE REVIEW

File: packages/gatsby-theme-bodiless/create-node.js [hunk 1]
```diff
@@ -243,12 +243,17 @@ const fixed = async ({
   };
 };
 
-const generateGatsbyImage = async ({ file, preset, reporter }, options) => {
+const generateGatsbyImage = async ({
+  file, preset, reporter, pathPrefix
+}, options) => {
+  console.log('pp', pathPrefix);
```
Comment (dewen) [inline packages/gatsby-theme-bodiless/create-node.js:249]: Remove debug line.

File: sites/__cxstarter__/gatsby-config.js [hunk 2]
```diff
@@ -116,5 +116,6 @@ module.exports = {
   flags: {
     DEV_SSR: false,
   },
+  pathPrefix: process.env.GATSBY_PATH_PREFIX,
```
Comment (dewen) [inline sites/__cxstarter__/gatsby-config.js:119]: default to empty string?

`pathPrefix: process.env.GATSBY_PATH_PREFIX,`
=>
`pathPrefix: process.env.GATSBY_PATH_PREFIX || ''`,

Comment (wodenx) [reply sites/__cxstarter__/gatsby-config.js:119]:
```suggestion
  pathPrefix: process.env.GATSBY_PATH_PREFIX || '',
```

File: packages/gatsby-theme-bodiless/create-node.js (hunk 1 above)
Comment (wodenx) [reply packages/gatsby-theme-bodiless/create-node.js:249]:
```suggestion
```

**Review by lucas-varela — APPROVED** [review]

**Pull request merged**